driver/src/io_backend/ext/xaie_io_common.c

```diff
@@ -433,6 +433,7 @@ AieRC _XAie_RequestRscCommon(XAie_DevInst *DevInst, XAie_BackendTilesRsc *Args)
 			Args->Rscs[j].RscId = RscArrPerTile[j];
 		}
 	}
+    free((void*)RscArrPerTile);
```

Comment:
2 problems:
* you don't free in every code path
* indentation

Reply:
right...